citesphere-model/src/main/java/edu/asu/diging/citesphere/data/bib/impl/CitationMongoDao.java

```diff
@@ -59,6 +59,7 @@
         Query query = new Query();
         query.addCriteria(Criteria.where("group").is(groupId));
         query.addCriteria(Criteria.where("collections").is(collectionId));
+        query.addCriteria(Criteria.where("itemType").ne(ItemType.ATTACHMENT.name()));
```

Comment:
Please also add the exclusion of notes here (as in `findCitations`). It's probably a bug that it isn't included. It's not part of this story but might as well fix it.